rules/docker-snmp-sv2.mk

```diff
@@ -14,3 +14,4 @@ $(DOCKER_SNMP_SV2)_RUN_OPT += --net=host --privileged -t
 $(DOCKER_SNMP_SV2)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 # mount Arista platform python libraries to support corresponding platforms SNMP power status query
 $(DOCKER_SNMP_SV2)_RUN_OPT += -v /usr/lib/python3/dist-packages/arista:/usr/lib/python3/dist-packages/arista:ro
+$(DOCKER_SNMP_SV2)_RUN_OPT += -v /usr/lib/python3/dist-packages/smbus.cpython-35m-x86_64-linux-gnu.so:/usr/lib/python3/dist-packages/smbus.cpython-36m-x86_64-linux-gnu.so
```

Comment:
>DOCKER_SNMP_SV2 [](start = 2, length = 15)

Sorry for missing the context.
Why snmp docker needs smbus? which component needs? for what purpose?

Reply:
On the 7060CX-32S we have to query the power supply status from an smbus device.
We therefore have to use the smbus module to do such query.

Snmp currently loads the platform plugins to provide power supply information, without this fix snmp-agent will throw a backtrace due to the missing module.
Hopefully this is just a temporary fix until STATE_DB contains the power supply information and snmp makes use of it rather than calling into a platform specific script.

Reply:
Could you add comment like:
# Only for Arista 7060CX-32
# temporary workaround to allow snmp container to import smbus for arista

---
In reply to: [274726238](https://github.com/Azure/sonic-buildimage/pull/2772#discussion_r274726238) [](ancestors = 274726238)